src/components/RecipeCard.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { TimeIcon } from '@chakra-ui/icons';
+import Img from 'gatsby-image';
+import { graphql } from 'gatsby';
+
+interface Props {
+  data: GatsbyTypes.RecipeCardFragment;
+}
+
+function RecipeCard({ data }: Props): JSX.Element {
+  return (
+    <div style={{padding: '50px 10px 50px 10px', marginRight: '25px'}}>
+      <div style={{ marginBottom: '5%'}}>
+        <h3
+          style={{
+            position: 'absolute',
+            fontWeight: 850,
+            textAlign: 'left',
+            fontSize: 14,
+          }}
+        >
+          {data.title}
+        </h3>
+        <TimeIcon
+          style={{ marginTop: 3, color: '#65BF73', position: 'absolute', marginLeft: '78%'}}
```

Comment:
Also for coloring, we can pass `color='green'` as a prop (we defined `"green"` to match our HF-HF theme color in [theme.ts](https://github.com/GTBitsOfGood/healthy-families-healthy-futures/blob/main/src/%40chakra-ui/gatsby-plugin/theme.ts#L5)